addons/website/static/tests/tours/website_form_editor.js

```diff
@@ -90,7 +100,8 @@ odoo.define('website.tour.form_editor', function (require) {
         }
         if (type !== 'checkbox' && type !== 'radio' && type !== 'select') {
             let inputType = type === 'textarea' ? type : `input[type="${type}"]`;
-            testText += `:has(${inputType}[name="${name}"]${required ? '[required]' : ''})`;
+            const nameAttribute = isCustom && label ? getQuotesEncodedName(label) : name;
+            testText += `:has(${inputType}[name="${nameAttribute}"]${required ? "[required]" : ""})`;
```

Comment:
I don't understand this part of the diff, why is it needed now? Could you explain?
Isn't this something which was actually needed regardless of your diff, as this code was actually not working in the case of an custom field being an input with a label?

Reply:
It is only needed now because before this diff, `addCustomField` was always called with a `type` argument that was either `checkbox`, `radio` or `select`. Because of that, this part of the code was never reached in the case of a `custom Field` (see the`if` condition -> `if (type !== 'checkbox' && type !== 'radio' && type !== 'select')`)

Reply:
Ok, that's what I thought, then this should really ideally be in a separate commit (sometime even worth doing in an earlier version), so the PR is easier to read and your actual fix in not noised by other fixes.

Reply:
It is now fixed :+1: